third_party/terraform/resources/resource_compute_instance.go.erb

```diff
@@ -611,7 +611,26 @@ func resourceComputeInstance() *schema.Resource {
 					},
 				},
 			},
-
+<% unless version == 'ga' -%>
+			"confidential_instance_config": {
+				Type:     schema.TypeList,
+				MaxItems: 1,
+				Optional: true,
+				ForceNew: true,
+				Computed: true,
+				Description: `The Confidential VM config being used by the instance.  on_host_maintenance has to be set to TERMINATE or this will fail to create.`,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"enable_confidential_compute": {
+							Type:         schema.TypeBool,
+							Optional:     true,
```

Comment:
Can we make this required so that if the user defines the `confidential_instance_config` block, they must also define this value?

Reply:
fixed